deepchem/models/torch_models/tests/test_highway.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+try:
+    import torch
+    import numpy as np
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+    pass
+from deepchem.models.torch_models.layers import Highway
```

Comment:
this line should be inside the test functions as it will run as a part of all the tests

Reply:
Resolved